docs/source/common_issues.rst

```diff
@@ -417,14 +417,14 @@ Example:
    # The rest of this file doesn't apply to Windows.
 
 Some other expressions exhibit similar behavior; in particular,
-``typing.TYPE_CHECKING``, variables named ``MYPY``, and any variable
+:py:data:`~typing.TYPE_CHECKING`, variables named ``MYPY``, and any variable
```

Comment:
The document mentions the `TYPE_CHECKING` constant sometimes as `TYPE_CHECKING` and sometimes as `typing.TYPE_CHECKING`. The occurrence of `typing.XXX` throughout the docs is marginal and the link leads to the `typing` module docs anyway, so I decided to drop the `typing.` prefix.

Reply:
+1